packages/extension/src/view/devtools/components/tabContent/cookies/components/cookieDetails/components/details.tsx

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Maybe there isn't a need for a components folder, as we already in component folder. It can be inside So `components/details.tsx` => `details/index.tsx`. Isn't there a test for it?

Reply:
Tests for this are inside `cookies/tests/cookieTab.tsx)`